.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.ignoreWords": ["noopener", "noreferrer", "peoplemakeglasgow"]
+}
```

Comment:
please don't submit your IDE configuration in this PR. You can always add it to .gitignore

Reply:
Hi Mozart, thanks for this feedback. I am not quite sure if I understood this. Could you explain me how to do this?
Thanks